tests/swarmtest/swarmtest.go

```diff
@@ -20,20 +22,27 @@ type SwarmTest struct {
 	NumWorkers int
 	Compile    func() error
 	Run        func() error
+	NumJobs    int
+	Timeout    time.Duration
+	mutex      *sync.Mutex
```

Comment:
Running go test -race ./... returns an error because there is access from multiple go routines to the cmd objects.  

I assume this is safe because its processes, but to make our test suite happy I added the mutex